lib/src/components/chat/ds_message_bubble.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:blip_ds/src/theme/systems_colors.dart';
+import 'package:flutter/material.dart';
+
+class DSMessageBubble extends StatelessWidget {
+  final bool alignRight;
+  final Widget child;
+  final EdgeInsets contentPadding;
+  final Map<String, bool> borderRadius;
+
+  const DSMessageBubble({
+    Key? key,
+    required this.alignRight,
+    required this.child,
+    this.contentPadding = const EdgeInsets.symmetric(
+      vertical: 10.0,
+      horizontal: 14.0,
+    ),
+    this.borderRadius = const {
+      'topLeft': false,
+      'topRight': false,
+      'bottomLeft': false,
+      'bottomRight': false
+    },
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Color containerColor = alignRight
```

Comment:
It's not better to create all of this inside constructor, instead of build method? Because some of this variables don't change unless the constructor params change.